src/pretix/base/models/organizer.py

```diff
@@ -57,7 +58,8 @@ def settings(self) -> SettingsProxy:
         """
         Returns an object representing this organizer's settings
         """
-        return SettingsProxy(self, type=OrganizerSetting)
+        from pretix.control.forms.global_settings import GlobalSettingsObject
```

Comment:
Yeah, I believe we need to move that to ``pretix.base.settings``. Importing ``control`` from ``base`` should be avoided, just as you suggested.